### DISCUSSION TITLE
how to do routing with hostname or host Header value?

### DISCUSSION THREAD

**Discussion by 0xPratham** [elysiajs/elysia#955] [Q&A]
something similar like this -> [https://hono.dev/docs/api/routing#routing-with-hostname](https://hono.dev/docs/api/routing#routing-with-hostname)

**Comment by DIEGOHORVATTI** [comment]
@0xPratham  Can you explain your doubts or what you intend to do better?

**Reply by 0xPratham** [reply]
I am trying to use multiple domains with the same API and want to identify which domain the API is called from. To achieve this, I plan to route the request based on the domain.
